collector/procpath.go

```diff
@@ -0,0 +1,17 @@
+package collector
```

Comment:
Wanted to comment about license headers, but seems like they are missing everywhere.